Prefabs/Locomotion/DestinationLocations/SharedResources/Scripts/DestinationLocationLockedStateTag.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace VRTK.Prefabs.Locomotion.DestinationLocations
+{
+    using UnityEngine;
+
+    public class DestinationLocationLockedStateTag : MonoBehaviour
+    {
+        public virtual void Start() { }
```

Comment:
I can't find it anymore, but I think we have/had a comment on another empty method like this somewhere to explain that we only do implement this as an empty method to allow showing the enabled checkbox in the inspector. Maybe we do this here, too?